### PR TITLE
RRD-34: Create player on server

### PR DESCRIPTION
**Pull request by martinraveglia**
- Update shared inputs logic
- Update CreateAccount layout
- Update interfaces and enums folder structure

### CODE REVIEW

File: src/components/shared/input/index.tsx [hunk 1]
```diff
@@ -1,17 +1,44 @@
...
+  type = 'text',
+  autocapitalize = 'none',
+  rightElement,
+}: Props<TFormValues>) => {
   return (
     <View style={styles.containerInput}>
       <Controller
         control={control}
-        // rules={{
-        //   required: true,
```
Comment (RadiumDraft) [inline src/components/shared/input/index.tsx:13]: Its necessary this comment?

File: src/screens/createAccount/index.tsx [hunk 2]
```diff
@@ -1,40 +1,67 @@
...
   lastName: string;
   email: string;
   password: string;
   repeatPassword: string;
+  skill: string;
+  position: PlayerPosition;
 }
 
 const CreateAccount = () => {
+  const dispatch = useDispatch();
```
Comment (RadiumDraft) [inline src/screens/createAccount/index.tsx:27]: We should create first the local state whit useState and below the selectors and custom hooks

File: src/screens/createAccount/index.tsx [hunk 3]
```diff
@@ -84,13 +109,48 @@ const CreateAccount = () => {
...
       />
+      <Input
+        control={control}
+        name="skill"
+        placeholder="Skill Score"
+        onFocus={() => onFocusInput('skill')}
+        error={errors.skill?.message}
+        rightElement={
+          <View m={2}>
+            <Popover trigger={props => <InfoIcon {...props} />}>
```
Comment (RadiumDraft) [inline src/screens/createAccount/index.tsx]: We could create a shared component for the Popover and move all this to another file

**Review by RadiumDraft — CHANGES_REQUESTED** [review]

File: src/components/shared/input/index.tsx (hunk 1 above)
Comment (martinraveglia) [reply src/components/shared/input/index.tsx:13]: I don't think so, actually, I've removed the comment on this PR

File: src/screens/createAccount/index.tsx (hunk 3 above)
Comment (martinraveglia) [reply src/screens/createAccount/index.tsx]: Done!

**Review by julianv97 — APPROVED** [review]

**Review by RadiumDraft — APPROVED** [review]

**Pull request merged**